donald_wade/week_01/004/main.js

```diff
@@ -0,0 +1,299 @@
+// homework
+// https://gist.github.com/wofockham/8ac3c1d747f345d89d3d
+
+// create an object containing each line as an array.
+var subway = {
+  "LN" : ["Times Square", "34th Street", "28th Street", "23rd Street", "Union Square", "8th Avenue"],
+  "LL" : ["8th Avenue", "6th Avenue", "Union Square", "3rd Avenue", "1st Avenue"],
+  "L6" : ["Grand Central", "33rd Street", "28th Street", "23rd Street", "Union Square", "Astor Place"],
+}
+
+var error_message = "INPUT ERROR";
+
+// this is the main driving function in program; it calls the other functions
+function selectorFunction() {
```

Comment:
You might want to break this huge function into smaller ones, that do something one. For example function that just takes input, function that check if we need to throw an error etc. In this way it ll be more readable and more reusable.